terra_layer/models.py

```diff
@@ -160,6 +160,9 @@ class Layer(models.Model):
 
     layer_style = JSONField(default=dict)
     layer_style_wizard = JSONField(default=dict)
```

Comment:
Unused fields ?

Reply:
Yes, but i keep them until the next delivery to keep data if the migration is broken for some reason